src/backend/sideload/games.ts

```diff
@@ -0,0 +1,232 @@
+import {
+  GameSettings,
+  GameInfo,
+  SideloadGame,
+  InstalledInfo
+} from '../../common/types'
+import { libraryStore } from './electronStores'
+import { GameConfig } from '../game_config'
+import {
+  isWindows,
+  isMac,
+  isLinux,
+  execOptions,
+  heroicGamesConfigPath
+} from '../constants'
+import { execAsync, killPattern, notify } from '../utils'
+import { logError, logInfo, LogPrefix, logWarning } from '../logger/logger'
+import { dirname, join } from 'path'
+import { constants as FS_CONSTANTS, existsSync, rmSync } from 'graceful-fs'
+import i18next from 'i18next'
+import { runWineCommand, setupEnvVars } from '../launcher'
+import { access, chmod } from 'fs/promises'
+import { addShortcuts, removeShortcuts } from '../shortcuts/shortcuts/shortcuts'
+
+export function appLogFileLocation(appName: string) {
+  return join(heroicGamesConfigPath, `${appName}-lastPlay.log`)
+}
+
+export function getAppInfo(appName: string): GameInfo {
+  const store = libraryStore.get('games', []) as GameInfo[]
+  return store.filter((app) => app.app_name === appName)[0] || {}
+}
+
+export async function getAppSettings(appName: string): Promise<GameSettings> {
+  return (
+    GameConfig.get(appName).config ||
+    (await GameConfig.get(appName).getSettings())
+  )
+}
+
+export function addNewApp({
```

Comment:
I wonder if this one along with the `removeApp` could be done on the frontend instead of backend. Would be less code to achieve the same.

Reply:
Maybe it's better to leave that here in the backend? for consistency with the other stores, I think it can be confusing to handle similar things some in the backend and some in the frontend

Reply:
Yes, will keep it here 👍🏽